web/template/team-work-in-progress.gotmpl

```diff
@@ -43,19 +43,27 @@
                 <label class="govuk-label" for="sort">
                   Show caseworker data for
                 </label>
-                <select class="govuk-select">
-                  <option>New cases worked today</option>
+                <select class="govuk-select" data-select-show>
+                  <option value="cases-worked">New cases worked today</option>
+                  <option value="tasks-completed">Tasks completed today</option>
                 </select>
               </div>
             </div>
             <div class="govuk-grid-column-three-quarters">
```

Comment:
I was going to suggest using `aria-controls`/`id` rather than `value`/`data-select-id`, but I can't find any evidence of it playing nicely when used on `<option>`. Just adding `aria-live` at least lets screenreader users know what has changed.
```suggestion
            <div class="govuk-grid-column-three-quarters" aria-live="polite">
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#basic_example_dropdown_box_updates_useful_onscreen_information I love when they have an example of the same scenario